### ISSUE TITLE
[Feaure idea] "Classic" money mode for custom scenarios

### ISSUE THREAD

**Issue by rctdude2** [IntelOrca/OpenRCT2#1000]
As the title says, there would be an option when creating custom scenarios to enable the "classic" money mode for that scenario which would allow the player to set costs on both rides and the park entrance, like in the original RCT. I personally liked this feature, and I think it would be useful for the RCT1 scenario ports for RCT2 where it was enabled.

**Comment by Gymnasiast** [comment]
Yes, it would be a nice feature. Do keep in mind it can only be implemented once OpenRCT2 switches to its own save format, since it has to maintain compatibility with SC6/SV6 in the meantime.

**Comment by PFCKrutonium** [comment]
Perhaps, perhaps not - if you switch the mode and put a price on the park entrance, then switch back, you can effectively use both to make money. So maybe a cheat to just open both for now?

**Comment by pizza2004** [comment]
Yeah, the only real reason you can't do it is no way to tell the game to leave both open, but I imagine there should be some sort of way of doing that without breaking compatibility of the save file structure with vanilla RCT2.

**Comment by IntelOrca** [comment]
We are not ready to implement this feature right now, but it will eventually be implemented along with all other RCT1 features.

**Comment by Horsti12** [comment]
This is now possible with a cheat code, so it can be closed.

**Comment by pizza2004** [comment]
I disagree.  I think that this can't be fully considered implemented until we have our own save format and the ability to specify new scenarios with a different setting.  No game feature or scenario should ever have to rely on cheats to function.

**Comment by Gymnasiast** [comment]
The plan is to provide three modes:
* Pay per ride and free entry
* Pay for entry and free rides
* Pay for entry and rides

The current cheat will probably be replaced with a way to switch this mode.

**Comment by IntelOrca** [comment]
Or two checkboxes (4 modes)
* [ ] Free entrance
* [ ] Free rides

...allowing very difficult parks where you can not charge for entrance or rides.

**Comment by Patrik356b** [comment]
Checkbox option I would much prefer
